cumulus_library/studies/core/builder_documentreference.py

```diff
@@ -35,7 +36,31 @@ def prepare_queries(
                     column_name="type",
                     is_array=False,
                     target_table="core__documentreference_dn_type",
-                )
+                ),
+                # TODO: The spec allows any value for category, but it's unclear what
+                # the value for the systems would be. For the time being, we select the
+                # spec valid code (which is a set of one value).
+                # It may be worth confirming the values in this field with either the
+                # quality or discovery studies on an ongoing basis to find other valid
+                # values
```

Comment:
I might reword this slight for clarity:
- instead of spec, maybe specifically mention the US Core v4 profile (vs the base FHIR spec, which doesn't care at all about the category system)
- maybe instead of "allows any" mention "is extensible" (or if that's too nerdy, something about "allows any, but prefers this one")
- for "spec valid code" maybe say something like "the recommended code system"
- when I hear "is a set of one value", I just wanna know the one value :smile: - maybe mention `one value: "clinical-note"`

Reply:
yep, all great notes.